umap/static/umap/js/umap.autocomplete.js

```diff
@@ -130,7 +130,7 @@ L.U.AutoComplete = L.Class.extend({
 
   onBlur: function () {
     var self = this
-    setTimeout(function () {
+    setTimeout(() => {
```

Comment:
Couldn't we leave out the anonymous function totally at the end ?

Reply:
Yup, let's do it together, I prefer that we check that the feature still works once we rewrite these parts.